install-jellyfin.ps1

```diff
@@ -51,33 +51,48 @@ else
 
 if($Quiet.IsPresent -or $Quiet -eq $true){
     $Script:JellyfinDataDir = "$env:AppData\jellyfin\"
-    if([string]::IsNullOrEmpty($InstallLocation)){$Script:DefaultJellyFinInstallDirectory = "$env:Appdata\jellyfin\"}else{$Script:DefaultJellyFinInstallDirectory = $InstallLocation}
+    if([string]::IsNullOrEmpty($InstallLocation)){
+        $Script:DefaultJellyfinInstallDirectory = "$env:Appdata\jellyfin\"
+    }else{
+        $Script:DefaultJellyfinInstallDirectory = $InstallLocation
+    }
     
-    if([string]::IsNullOrEmpty($EmbyLibraryLocation)){$Script:defaultEmbyDataDir = "$env:Appdata\Emby-Server\data\"}else{$Script:defaultEmbyDataDir = $EmbyLibraryLocation}
+    if([string]::IsNullOrEmpty($EmbyLibraryLocation)){
+        $Script:defaultEmbyDataDir = "$env:Appdata\Emby-Server\data\"
+    }else{
+        $Script:defaultEmbyDataDir = $EmbyLibraryLocation
+    }
     
-    if($InstallAsService.IsPresent -or $InstallAsService -eq $true){$Script:InstallAsService = $true ; $Script:JellyfinDataDir = "C:\WINDOWS\system32\config\systemprofile\AppData\Roaming\jellyfin\" }else{$Script:InstallAsService = $false}
-    if($null -eq $ServiceUser){$Script:InstallServiceAsUser = $false}else{$Script:InstallServiceAsUser = $true; $Script:UserCredentials = $ServiceUser;  $Script:JellyfinDataDir = "C:\Users\$($Script:UserCredentials.UserName)\Appdata\Roaming\jellyfin\"}
+    if($InstallAsService.IsPresent -or $InstallAsService -eq $true){
+        $Script:InstallAsService = $true 
+        $Script:JellyfinDataDir = "C:\WINDOWS\system32\config\systemprofile\AppData\Roaming\jellyfin\" 
```

Comment:
Windows might not be installed to `C:\Windows`, at least use `%WINDIR%` (and better use some PowerShell version of [GetSystemDirectory](https://docs.microsoft.com/en-us/windows/desktop/api/sysinfoapi/nf-sysinfoapi-getsystemdirectorya) if possible).

Reply:
Don't assume C:\ exists.